src/test/java/de/eposcat/master/PageTest.java

```diff
@@ -0,0 +1,206 @@
+package de.eposcat.master;
+
+
+import de.eposcat.master.approachImpl.EAV_DatabaseAdapter;
+import de.eposcat.master.approachImpl.IDatabaseAdapter;
+import de.eposcat.master.connection.H2ConnectionManager;
+import de.eposcat.master.connection.RelationalApproach;
+import de.eposcat.master.exceptions.BlException;
+import de.eposcat.master.model.Attribute;
+import de.eposcat.master.model.AttributeBuilder;
+import de.eposcat.master.model.AttributeType;
+import de.eposcat.master.model.Page;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+public class PageTest {
+
+    static IDatabaseAdapter dbAdapter;
+    static Attribute defaultAttribute;
+
+    @BeforeAll
+    static void initDataBase() {
+        H2ConnectionManager connectionManager = new H2ConnectionManager(RelationalApproach.EAV);
```

Comment:
This database setup will be needed in other test classes and can be extracted in a super **Test** class to avoid code duplication. This refactoring can be done a separate PR.